src/MatrixExp_Hamiltonians.jl

```diff
@@ -318,84 +318,52 @@ function propagator_linear_inversion_pulse(ω1, T, B1, R2s, _, dR2sdB1, grad_typ
 end
 
 function z_rotation_propagator(rfphase_increment, _)
-    u_rot = @SMatrix [cos(rfphase_increment) -sin(rfphase_increment) 0 0 0 0;
-                      sin(rfphase_increment)  cos(rfphase_increment) 0 0 0 0;
-                                     0                0  1 0 0 0;
-                                     0                0  0 1 0 0;
-                                     0                0  0 0 1 0;
-                                     0                0  0 0 0 1]
+	(sϕ,cϕ) = sincos(rfphase_increment)
```

Comment:
Do you mind removing the brackets and adding a whitespace `sϕ, cϕ = sincos(rfphase_increment)`, and aligning the line with other code?

Reply:
Yes, of course, I do not mind. :-) I am a hard tab aficionado and forgot to enable spaces.